articles/toc.yml

```diff
@@ -1,81 +1,16 @@
-    href: overview/intro.md
-  - name: Why Intent Architect
-    href: overview/why_intent_architect.md
-  - name: FAQ
-    href: getting_started/faq.md
-- name: Getting Started
-  homepage: getting_started/create_an_aspnetcore_web_app.md
-  items:
-  - name: Create an ASP.NET Core web app
-    href: getting_started/create_an_aspnetcore_web_app.md
-  - name: Create your own Module
-    href: getting_started/create_your_own_module.md
-- name: Modules
-  homepage: modules/overview.md
-  items:
-  - name: Overview
-    href: modules/overview.md
-  - name: IModSpec File
-    href: modules/imodspec_file.md
-  - name: Templates
-    href: templates/overview.md
-  - name: Factory Extensions
-    href: modules/factory_extensions.md
-- name: Templates
-  homepage: templates/overview.md
-  items:
-  - name: Overview
-    href: templates/overview.md
-  - name: The Template Class
-    href: templates/the_template_class.md
-  - name: Registration
-    href: templates/registrations.md
-  - name: Configuration
-    href: templates/configuration.md
-  - name: Code Output
-    href: templates/code_output.md
-  - name: Extensions
-    href: templates/extensions.md
-- name: Core Modules
-  homepage: core_modules.md
-  items:
-  - name: Intent Common
-    href: ../modules/intent_common/overview.md
-  - name: Intent Common Types
-    href: ../modules/intent_common_types/overview.md
-  - name: Intent Nuget Installer
-    href: ../modules/intent_nuget_installer/overview.md
-  - name: Intent Visual Studio Projects
-    href: ../modules/intent_visual_studio_projects/overview.md
-  - name: Roslyn Weaver 
-    href: ../modules/roslyn_weaver/
-- name: User Manual
-  items:
-  - name: The shell
-    href: user_manual/shell.md
-  - name: Home screen
-    href: user_manual/home_screen.md
-  - name: Creating a Solution
-    href: user_manual/solution_screen.md
-  - name: Creating an Application
-    href: user_manual/new_application_wizard.md
-  - name: Application home screen
-    href: user_manual/application_screen.md
-  - name: Application Configuration
-    href: user_manual/application_configuration.md
-  - name: Application Modules
-    href: user_manual/application_modules.md
-  - name: Model your Domain
-    href: user_manual/domain.md
-  - name: Model your Data Contracts
-    href: user_manual/dto.md
-  - name: Model your Services
-    href: user_manual/services.md
-  - name: Push the play button
-    href: user_manual/software_factory_execution.md
-  - name: Repository Configuration
-    href: user_manual/repository_configuration.md
\ No newline at end of file
+  href: overview/index.md
+- name: Tutorials
+  topicHref: tutorials/index.md
+  href: tutorials/toc.yml
+- name: How-to guides
+  href: howtos/toc.yml
+  topicHref: howtos/index.md
+- name: Reference documentation
+  topicHref: references/index.md
+  href: references/toc.yml
+- name: Topic guides
+  topicHref: topics/index.md
+  href: topics/toc.yml
+- name: Getting help
+  href: help/toc.yml
```

Comment:
For maintainability purposes, is it possible for the `name`s to be blank and have DocFX automatically populate it with the article title from an `xref`? This may require that we change TOC files from `.yml` to `.md`.